src/test/java/org/opensearch/ad/caching/PriorityCacheTests.java

```diff
@@ -1,655 +1,657 @@
-            cacheProvider.get(entity1.getModelId(detectorId).get(), detector);
-        }
-
-        // put modelState5 in dedicated and modelState6 in shared cache
-        when(memoryTracker.canAllocate(anyLong())).thenReturn(true);
-        cacheProvider.hostIfPossible(detector, modelState1);
-        cacheProvider.hostIfPossible(detector, modelState2);
-
-        // two entities get inserted to cache
-        assertTrue(null != cacheProvider.get(entity1.getModelId(detectorId).get(), detector));
-        assertTrue(null != cacheProvider.get(entity2.getModelId(detectorId).get(), detector));
-
-        Entity entity5 = Entity.createSingleAttributeEntity("attributeName1", "attributeVal5");
-        when(memoryTracker.memoryToShed()).thenReturn(memoryPerEntity);
-        for (int i = 0; i < 2; i++) {
-            // bypass doorkeeper, CacheBuffer created, and trigger clearMemory
-            cacheProvider.get(entity5.getModelId(detectorId2).get(), detector2);
-        }
-
-        assertTrue(null != cacheProvider.get(entity1.getModelId(detectorId).get(), detector));
-        // entity 2 removed
-        assertTrue(null == cacheProvider.get(entity2.getModelId(detectorId).get(), detector));
-        assertTrue(null == cacheProvider.get(entity5.getModelId(detectorId2).get(), detector));
-    }
-
-    public void testSelectEmpty() {
-        Collection<Entity> cacheMissEntities = new ArrayList<>();
-        cacheMissEntities.add(entity1);
-        Pair<List<Entity>, List<Entity>> selectedAndOther = cacheProvider.selectUpdateCandidate(cacheMissEntities, detectorId, detector);
-        assertEquals(0, selectedAndOther.getLeft().size());
-        assertEquals(0, selectedAndOther.getRight().size());
-    }
-
-    // test that detector interval is more than 1 hour that maintenance is called before
-    // the next get method
-    public void testLongDetectorInterval() {
-        when(clock.instant()).thenReturn(Instant.ofEpochSecond(1000));
-        when(detector.getDetectionIntervalDuration()).thenReturn(Duration.ofHours(12));
-        String modelId = entity1.getModelId(detectorId).get();
-        // record last access time 1000
-        cacheProvider.get(modelId, detector);
-        assertEquals(-1, cacheProvider.getLastActiveMs(detectorId, modelId));
-        // 2 hour = 7200 seconds have passed
-        long currentTimeEpoch = 8200;
-        when(clock.instant()).thenReturn(Instant.ofEpochSecond(currentTimeEpoch));
-        // door keeper should not be expired since we reclaim space every 60 intervals
-        cacheProvider.maintenance();
-        // door keeper still has the record and won't blocks entity state being created
-        cacheProvider.get(modelId, detector);
-        // * 1000 to convert to milliseconds
-        assertEquals(currentTimeEpoch * 1000, cacheProvider.getLastActiveMs(detectorId, modelId));
-    }
-}
+// @anomaly-detection.create-components. Commented until we have support for SDKClusterSettings for extensions
```

Comment:
This commenting format totally loses indentation and makes tracking the diff harder when it's uncommented.

Use format-preserving comment style:
```java
/*-
  When you use a hyphen like this
    the indentation
      is preserved just as it was
    so the diff is nice
  when you eventually uncomment
*/
```